### ISSUE TITLE
systemd-fsck: fsck failed

### ISSUE THREAD

**Issue by kouros17** [systemd/systemd#12135]
I'm using **manjaro** and systemd **241.1023-1**
From journalctl I get:

`manjaro systemd-fsck[718]: fsck failed with exit status 16.`

**Comment by poettering** [comment]
please provide a longer log excerpt

**Comment by kouros17** [comment]
**journalctl -b | grep fsck**

```
Μαρ 28 16:18:39 manjaro systemd-fsck[718]: Usage: fsck.ext4 [-panyrcdfktvDFV] [-b superblock] [-B blocksize]
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:                 [-l|-L bad_blocks_file] [-C fd] [-j external_journal]
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:                 [-E extended-options] [-z undo_file] device
Μαρ 28 16:18:39 manjaro systemd-fsck[718]: Emergency help:
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -p                   Automatic repair (no questions)
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -n                   Make no changes to the filesystem
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -y                   Assume "yes" to all questions
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -c                   Check for bad blocks and add them to the badblock list
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -f                   Force checking even if filesystem is marked clean
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -v                   Be verbose
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -b superblock        Use alternative superblock
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -B blocksize         Force blocksize when looking for superblock
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -j external_journal  Set location of the external journal
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -l bad_blocks_file   Add to badblocks list
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -L bad_blocks_file   Set badblocks list
Μαρ 28 16:18:39 manjaro systemd-fsck[718]:  -z undo_file         Create an undo file
Μαρ 28 16:18:39 manjaro systemd-fsck[718]: fsck failed with exit status 16.
Μαρ 28 16:18:39 manjaro systemd-fsck[718]: Ignoring error.
Μαρ 28 16:18:39 manjaro audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-fsck@dev-disk-by\x2duuid-378b1275\x2ddf1b\x2d4c66\x2db067\x2da2c0566b67c3 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Μαρ 28 16:18:39 manjaro kernel: audit: type=1130 audit(1553782719.142:6): pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-fsck@dev-disk-by\x2duuid-378b1275\x2ddf1b\x2d4c66\x2db067\x2da2c0566b67c3 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Μαρ 28 16:18:39 manjaro kernel: FAT-fs (nvme0n1p1): Volume was not properly unmounted. Some data may be corrupt. Please run fsck.
Μαρ 28 16:47:13 manjaro dbus-daemon[740]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.365' (uid=0 pid=12911 comm="sudo systemctl status systemd-fsck@.service ")
Μαρ 28 16:47:16 manjaro sudo[12911]:  giorgos : TTY=pts/0 ; PWD=/home/giorgos ; USER=root ; COMMAND=/usr/bin/systemctl status systemd-fsck@.service
Μαρ 28 16:47:58 manjaro dbus-daemon[740]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.371' (uid=0 pid=13211 comm="sudo systemctl status systemd-fsck ")
Μαρ 28 16:48:01 manjaro sudo[13211]:  giorgos : TTY=pts/0 ; PWD=/home/giorgos ; USER=root ; COMMAND=/usr/bin/systemctl status systemd-fsck
Μαρ 28 16:59:11 manjaro dbus-daemon[740]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.424' (uid=0 pid=17781 comm="sudo systemctl status systemd-fsck@.service ")
Μαρ 28 16:59:14 manjaro sudo[17781]:  giorgos : TTY=pts/1 ; PWD=/home/giorgos ; USER=root ; COMMAND=/usr/bin/systemctl status systemd-fsck@.service

```

**Comment by keszybz** [comment]
Just attach the whole output of `journalctl -b`.

**Comment by kouros17** [comment]
@keszybz it's very long

**Comment by kouros17** [comment]
[log.txt](https://github.com/systemd/systemd/files/3018791/log.txt)

**Comment by poettering** [comment]
hmm could you boot once with `systemd.log_level=debug systemd.log_target=kmsg log_buf_len=1M printk.devkmsg=on` added to your kernel cmdline, then provide "journalctl -b" output this results in?

**Comment by kouros17** [comment]
@poettering 
[journalctl -b](https://github.com/systemd/systemd/files/3019596/log.txt)

**Comment by poettering** [comment]
Hmm, I presume this is a version that includes 60ffa37a65a96c3af857a3dfc4a6fd47b20cc90e already? if so, I figure I know what the problem is.

**Comment by poettering** [comment]
Fix waiting in #12141. (At least my I am pretty sure that's the fix for your issue. Would be great if you could give it a whirl)

**Comment by philmmanjaro** [comment]
@kouros17: please test **241.1042-1** from our **unstable** branch.

**Comment by kouros17** [comment]
@philmmanjaro and @poettering it's fixed with **241.1042-1**!

**Comment by philmmanjaro** [comment]
@poettering: the more correct answer would be: I applied #12141
